### ISSUE TITLE
prefix_chars not supported

### ISSUE THREAD

**Issue by erasche** [erasche/argparse2tool#11]

**Comment by anton-khodak** [comment]
@mr-c argparse2tool supports prefix_chars, take a look at `test_prefixes` method https://github.com/erasche/argparse2tool/pull/32/commits/6f85899630c603d7c605e2bb5485d5d9312dbf6c

**Comment by anton-khodak** [comment]
or it regards galaxy part?

**Comment by mr-c** [comment]
Great; do we have a test for this? 

Looks like this old issue was accidently closed by GitHub during the recent merge; I re-opened it which is probably why you noticed it :-)

**Comment by anton-khodak** [comment]
that's what I pointed to :) https://github.com/erasche/argparse2tool/blob/eba1d7da12cda707c9a6ee76341f640185eacae8/test/test_arg2cwl.py#L251

**Comment by mr-c** [comment]
@anton-khodak Right, that's a test for `arg2cwl` I think we need a test on the Galaxy side
